src/Behavior/Binary.php

```diff
@@ -58,18 +58,6 @@ public function toDb($value, $key, $_)
             return $value;
         }
 
-        /**
```

Comment:
There is another comment and code regarding the duplicate transformation. Please verify whether `substr()` is still needed there.

Reply:
If this solved, please resolve the conversation